ground/src/main/java/com/google/android/ground/ui/home/HomeScreenViewModel.kt

```diff
@@ -98,4 +104,10 @@ internal constructor(
   fun showSyncStatus() {
     navigator.navigate(HomeScreenFragmentDirections.showSyncStatus())
   }
+
+  fun getUserData(): User? {
+    viewModelScope.launch { _userDetails.value = userRepository.getUserDetails() }
+    Timber.tag("getUserData()").w(":%s", userDetails.value?.email.toString())
+    return userDetails.value
```

Comment:
...so this value may not be populated by the time we get to this line. 

Instead, you can just inject `AuthenticationManager` and call `getAuthenticatedUser()` when you need it.